assets/queries/cloudFormation/BucketPolicyEnabled/test/negative.yaml

```diff
@@ -0,0 +1,66 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: A sample template
+Resources:
+  mybucket:
+    Type: "AWS::S3::Bucket"
+    DeletionPolicy: Retain
+    Properties:
+      ReplicationConfiguration:
+        Role:
+          "Fn::GetAtt":
+            - WorkItemBucketBackupRole
+            - Arn
+        Rules:
+          - Destination:
+              Bucket:
+                "Fn::Join":
+                  - ""
+                  - - "arn:aws:s3:::"
+                    - "Fn::Join":
+                        - "-"
+                        - - Ref: "AWS::Region"
+                          - Ref: "AWS::StackName"
+                          - replicationbucket
+              StorageClass: STANDARD
+            Id: Backup
+            Prefix: ""
+            Status: Enabled
+      VersioningConfiguration:
+        Status: Enabled
+      LoggingConfiguration:
+        DestinationBucketName: !Ref LoggingBucket
```

Comment:
Since `LoggingBucket `is not declared in this sample. Remove the `!Ref`, leaving only the string.